src/app/systelab-components/numpad/numpad.dialog.component.scss

```diff
@@ -0,0 +1,23 @@
+.nopadding {
+  padding: 2px !important;
+  margin: 0 !important;
+}
+.parentPadding {
+  padding: 15px;
+}
+.btn {
+  height: 55px;
+  line-height: 50px;
+  font-weight: bold;
+  font-size: 25px;
+}
+.container {
+  margin-right: auto;
+  margin-left: auto;
+}
+.inputNumpad{
+  height: 60px;
+  font-size: 42px;
+  line-height: 35px;
+  width: 100%;
+}
```

Comment:
Custom styles must start with slab- to differentiate from bootstrap styles. Do not overwrite bootstrap styles. Try yo use variables and dynamic heights when possible